src/app.js

```diff
@@ -1 +1,46 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios, {defaults} from 'axios'
+
+async function getCountryList() {
+    try {
+        const result = await axios.get('https://restcountries.com/v2/all?fields=name,flag,population,region')
+        let list = result.data
+        list.sort((a, b) => {
+            return a.population - b.population
+        })
```

Comment:
Dit had je nog iets korter kunnen schrijven
```suggestion
        list.sort((a, b) => a.population - b.population)
```